kernel/data-pipeline/scenario/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java

```diff
@@ -32,19 +35,17 @@ public final class Bootstrap {
      *
      * @param args args
      */
+    @SneakyThrows(InterruptedException.class)
     public static void main(final String[] args) {
         // Pay attention to the time zone, to avoid the problem of incorrect time zone, it is best to ensure that the time zone of the program is consistent with the time zone of the database server
-        // and mysql-connector-java 5.x version will ignore serverTimezone jdbc parameter and use the default time zone in the program
         // TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
-        StartCDCClientParameter parameter = new StartCDCClientParameter();
-        parameter.setAddress("127.0.0.1");
-        parameter.setPort(33071);
-        parameter.setUsername("root");
-        parameter.setPassword("root");
-        parameter.setDatabase("sharding_db");
-        parameter.setFull(true);
-        parameter.setSchemaTables(Collections.singletonList(SchemaTable.newBuilder().setTable("t_order").build()));
-        CDCClient cdcClient = new CDCClient(parameter, records -> log.info("records: {}", records));
-        cdcClient.start();
+        String address = "127.0.0.1";
+        CDCClientConfiguration clientConfig = new CDCClientConfiguration(address, 33071, records -> log.info("records: {}", records));
```

Comment:
When there's exception thrown on streaming, it should notify users, and then users could customize how to handle it, e.g. re-connect / re-subscribe / stop.